### ISSUE TITLE
Event Planning: general preparation & advertisement efforts

### ISSUE THREAD

**Issue by AARON-CLARK** [pharmaR/pharmaR#9]
Related to #2; wanted to create a separate issue to generalized some of the planning steps necessary to maximize attendance, especially when choosing dates for upcoming talks. Please feel free to edit if I missed some items.

## Proposal
**Each event will have a 1 month period of time dedicated to advertisement and preparation.**

### General prep
- [ ] Title & abstract composed
- [ ] Prepare slide deck
- [ ] Trial run on webinar platform (hopin/ zoom/ etc)

### Advertising
- [ ] Generate blog post for Pharmar.org
- [ ] R Validation Hub email blast, sharing Title & Abstract + fun graphic, maybe link to blog post
    - [ ] Send initial invitation + reminders at @ different intervals
        - [ ] 1 month out
        - [ ] 2 weeks out
        - [ ] 1 week out
        - [ ] 1 day before or morning of?
- [ ] Share title & abstract (and optionally, the blog post) with communication partners
    - [ ] R Consortium marketing team (Agueda Gama & Jesse Casman)
    - [ ] PHUSE - accessible through pharmaverse affiliation (marketing contacts to be determined by @AARON-CLARK)
    - [ ] Everyone: share posts created by our communication partners on social media
- [ ] Anything else?

**Comment by dgkf** [comment]
Thanks for organizing this @AARON-CLARK - I adapted most of this content to an "Event Planning Checklist" issue template and kicked off a pharmaR ["Events"](https://github.com/pharmaR/pharmaR/issues/9) project which will let us assign start and target dates to issues to get a little gantt chart view ([I added this issue just to test it out](https://github.com/orgs/pharmaR/projects/7))

**Comment by dgkf** [comment]
@pharmaR/ws-communications - Shall we organize a short call to walk through how the template and project board might be used? I think we can go through the (expedited) process for our upcoming all-hands. Even if there are initial hiccups in the process, I think it will be good to test it out and we can see how we can improve it from there.

**Comment by jmanitz** [comment]
> Thanks for organizing this @AARON-CLARK - I adapted most of this content to an "Event Planning Checklist" issue template and kicked off a pharmaR ["Events"](https://github.com/pharmaR/pharmaR/issues/9) project which will let us assign start and target dates to issues to get a little gantt chart view ([I added this issue just to test it out](https://github.com/orgs/pharmaR/projects/7))

Not sure why we need an additional tool. We have already an events repo and I had created respective issues for upcoming events: https://github.com/pharmaR/events. I would prefer to track things in a single location only, otherwise something is going to be out of synch all the time. 

For next weeks all-hands on, we can recycle the slides as discussed previously:
https://github.com/pharmaR/events/tree/main/R_Adoption_Series_2023

**Comment by jmanitz** [comment]
> Related to #2; wanted to create a separate issue to generalized some of the planning steps necessary to maximize attendance, especially when choosing dates for upcoming talks. Please feel free to edit if I missed some items.
> 
> ## Proposal
> **Each event should have a 1 month period of time dedicated to advertisement and preparation.** At a minimum, this means that a title & abstract have been created, registration details are provided, and the initial advertisement have been communicated approximately 30 prior to the event. When choosing an event date, we could use these criteria to help guide our selection.
> 
> ### General prep
> * [ ]  Title & abstract composed
> * [ ]  Prepare slide deck
> * [ ]  Trial run on webinar platform (hopin/ zoom/ etc)
> 
> ### Advertising
> * [ ]  R Validation Hub email blast, sharing Title & Abstract + fun graphic, maybe link to blog post
>   
>   * [ ]  Send initial invitation + reminders at @ different intervals
>     
>     * [ ]  1 month out
>     * [ ]  2 weeks out
>     * [ ]  1 week out
>     * [ ]  1 day before or morning of?
> * [ ]  Generate blog post for Pharmar.org
> * [ ]  Share title & abstract (and optionally, the blog post) with communication partners
>   
>   * [ ]  R Consortium marketing team (Agueda Gama & Jesse Casman)
>   * [ ]  PHUSE - accessible through pharmaverse affiliation (marketing contacts to be determined by @AARON-CLARK)
>   * [ ]  Everyone: share posts created by our communication partners on social media
> * [ ]  Anything else?

Thanks for setting this up. I am wondering what is the best time for a blog post and what content you imagine. 
Previously we for instance have a blog post if we plan a number of hands-on for the rest of the year or something like that. In addition we had a the discussion to have a blog post concurrent with the meeting/a few days later summarizing the contents.

**Comment by dgkf** [comment]
> <sup>[jmanitz](https://github.com/jmanitz) in https://github.com/pharmaR/pharmaR/issues/9#issuecomment-1561161241</sup>
> Not sure why we need an additional tool. We have already an events repo and I had created respective issues for upcoming events: https://github.com/pharmaR/events. I would prefer to track things in a single location only, otherwise something is going to be out of synch all the time.

Yep - I agree. The project board is org-wide, so it can aggregate issues from either repo. 

I wasn't sure if the events repo would be for cataloging presentations or for event planning. I was sort of under the impression that the scope was the former, but I'm fine with either interpretation as long as we're consistent.
